backend/src/main/java/ca/mcgill/purposeful/controller/IdeaController.java

```diff
@@ -26,12 +35,15 @@
  */
 
 @RestController
-@RequestMapping({"api/idea", "api/idea/"})
+@RequestMapping({"/api/idea", "/api/idea/"})
 public class IdeaController {
 
   @Autowired
   IdeaService ideaService;
 
+  @Autowired
+  RegularUserRepository regularUserRepository;
```

Comment:
Ideally, we shouldn't need any repositories to be auto wired into the controller - they should be used by the service methods

Reply:
Noted. I moved that work to the service method and will commit my changes soon.